components/service/glean/src/main/java/mozilla/components/service/glean/CommonMetricData.kt

```diff
@@ -39,6 +44,7 @@ interface CommonMetricData {
 
     companion object {
         internal const val DEFAULT_STORAGE_NAME = "default"
+        private const val JOB_TIMEOUT = 250L
```

Comment:
What's the time unit for this? Milliseconds? if so, let's rename this to `JOB_TIMEOUT_MS` (i.e. adding the `_MS` postfix)